deploy/charts/venafi-kubernetes-agent/README.md.gotmpl

```diff
@@ -6,134 +6,130 @@
 ## Additional Information
 
 The Venafi Kubernetes Agent connects your Kubernetes or OpenShift cluster to the Venafi Control Plane.
-You will require a Venafi Control Plane account to connect your cluster. 
-If you do not have you, you can sign up for a free trial now at:
+You will require a Venafi Control Plane account to connect your cluster.
+If you do not have one, you can sign up for a free trial now at:
 - https://venafi.com/try-venafi/tls-protect/
 
-Note that there are EU and US Venafi Control Plane options. 
+Note that there are EU and US Venafi Control Plane options.
 Upon signing up you will be redirected to one of either of the following login URLs:
-- https://ui.venafi.cloud/ (US) 
+- https://ui.venafi.cloud/ (US)
 - https://ui.venafi.eu/ (EU)
 
-## Installation:
+## Installation
 
-Using chart installation, there are two credentials required.
+The Helm chart is available from the following Venafi OCI registries:
 
-1) A registry credential to allow helm to pull the chart from our private OCI registry.
-2) A service account key pair used by the agent to authenticate to the Venafi Control Plane.
+- `oci://registry.venafi.cloud/charts/venafi-kubernetes-agent` (public)
+- `oci://private-registry.venafi.cloud/charts/venafi-kubernetes-agent` (private, US)
+- `oci://private-registry.venafi.eu/charts/venafi-kubernetes-agent` (private, EU)
 
-### 1) Setup registry credentials
+> Learn [how to access the private Venafi OCI registries](https://docs.venafi.cloud/vaas/k8s-components/th-guide-confg-access-to-tlspk-enterprise-components/).
```

Comment:
This is completely the right link, but what it links to is completely useless IMO.
Why would you have to enable this by talking to a human. Shouldn't this just be a feature enabled by default?

Surely being a Venafi Customer is enough to get access to a private revision of the `venafi-kubernetes-agent`?
![image](https://github.com/jetstack/jetstack-secure/assets/11051419/450e7d9d-2bf2-49c1-aec9-7bb0bb1a2b2f)

**No action needed,** just ranting.

Reply:
I raised an issue with the documentation team:
 * https://venafi.atlassian.net/browse/VC-28984

Reply:
This was added to the docs by PM in advance of the preview. The page will updated for the GA.